includes/discovery/os/hirschmann.inc.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
This file is an old file from a previous LibreNMS installation. I've removed it.